ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -672,6 +662,58 @@ open class BottomCommandingController: UIViewController, TokenizedControlInterna
 
     private var bottomBarBackgroundView: UIView?
 
+    // MARK: - CommandingItem and CommandingSection filtering
+
+    /// Filters the `heroItems` array for items with that are not hidden
+    private func updateVisibleHeroItems() {
+        let updatedVisibleHeroItems = heroItems.filter { item in
+            return !item.isHidden
+        }
+
+        visibleHeroItems = updatedVisibleHeroItems
+    }
+
+    /// Filters the `expandedListSections` 2D array for items with that are not hidden
+    private func updateVisibleExpandedListSections() {
+        /// Filter for all `CommandingSection`  that have at least 1 visible `CommandingItem`
+        let updatedVisibleExpandedListSections = expandedListSections.filter {$0.items.filter {!$0.isHidden}.count > 0}
```

Comment:
But maybe can split this line into separate lines anyway. I am likely misunderstanding, but feels like the outer filter is using implicit boolean conversion of whether the inner filter is not nil, but wouldn't that never be nil because it would just return an empty array instead?

So maybe should be 
```
expandedListSections.filter { expandedListSection in 
    return expandedListSection.items.contains { item in
        return !item.isHidden
    }
}
```

Reply:
Using `contains` has the potential for slightly better perf too!